### PR TITLE
add a messageStream via the options

### PR DESCRIPTION
**Pull request by laurenskling**
This would be nice in one of my projects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,16 @@ npm i strapi-provider-email-postmark
...
+| ----------------------------- | ------ | -------------------------------------------------------------------------------------------------- | -------- | --------- |
+| provider                      | string | The name of the provider you use                                                                                                                                              | yes      |           |
+| providerOptions               | object | Provider options                                                                                                                                                              | yes      |           |
+| providerOptions.apiKey        | string | Postmark API key. Please refer to [postmark docs](https://www.npmjs.com/package/postmark) for more                                                                            | yes      |           |
+| providerOptions.messageStream | string | Send through a specific message stream. Please refer to [postmark docs](https://postmarkapp.com/support/article/1207-how-to-create-and-send-through-message-streams) for more | no       | undefined |
```
Comment (ScottAgirs) [inline README.md]: Assuming (please correct me if I'm wrong) Message streams can change throughout the use within the app?

Should we call this `defaultMessageStream`?

File: lib/index.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ module.exports = {
           } = options;
 
           const msg = {
+            "MessageStream": settings.messageStream,
```
Comment (ScottAgirs) [inline lib/index.js]: Please consider adjusting to `"MessageStream": messageStream ||  settings.defaultMessageStream`.

Will also need to destructure `messageStream` from options on line 24 or so : )

**Comment by laurenskling** [comment]
@ScottAgirs any chance mate?

**Review by ScottAgirs — CHANGES_REQUESTED** [review]

**Comment by ScottAgirs** [comment]
> @ScottAgirs any chance mate?

@laurenskling Ugh, looks like I hadn't submitted the review, lol, sorry about that. 🤦🏼‍♂️ Please see the review, lmk what u think 🙌🏼

File: README.md (hunk 1 above)
Comment (laurenskling) [reply README.md]: I think that is very true

**Comment by laurenskling** [comment]
I made the changes, also the docs were different from the actual code. I moved the docs and code to use `settings` not `providerOptions`.

**Comment by ScottAgirs** [comment]
> I made the changes, also the docs were different from the actual code. I moved the docs and code to use `settings` not `providerOptions`.

Cool, this LGTM. Think we're all good here, if so you can expect a merge in a couple of hours. Thanks for the PR and commits : )

**Comment by laurenskling** [comment]
What do you say mate?

File: README.md [hunk 3]
```diff
@@ -41,6 +42,7 @@ module.exports = ({ env }) => ({
       apiKey: env("POSTMARK_API_KEY"),
     },
     settings: {
+      defaultMessageStream: env("POSTMARK_MESSAGE_STREAM"),
```
Comment (ScottAgirs) [inline README.md]: Can we pls remove this and add a hardcoded value? This adds an extra step, I'd prefer the example not to be opinionated in this regard.

**Review by ScottAgirs — CHANGES_REQUESTED** [review]
If we could make this minor tweak, it'll be good for a merge

File: CHANGELOG.md [hunk 4]
```diff
@@ -25,15 +32,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
-## [3.0.5] - 2020-09-03
-
-### Added
-
-- custom methods
-- templateId
-- templateAlias
-- variables
-
```
Comment (ScottAgirs) [inline CHANGELOG.md:36]: What this removed by mistake? 🤔

Comment (laurenskling) [reply CHANGELOG.md:36]: No, the 3.0.5 is doubled in the changelog, this one is wrong.

File: README.md (hunk 3 above)
Comment (laurenskling) [reply README.md]: done.

**Review by ScottAgirs — APPROVED** [review]
LGTM, thank you @laurenskling

**Comment by laurenskling** [comment]
Thanks mate

**Pull request merged**